src/components/Avatar/Avatar.test-int.js

```diff
@@ -0,0 +1,41 @@
+/* global Feature, Scenario */
+
+const KIND = 'Avatar.Avatar';
+
+Feature('Avatar');
+
+Scenario('Storybook Documentation', (I) => {
+    I.searchWithinIframe(KIND, 'Overview', () => {
+        I.see('Avatar: Avatar');
+        I.see('Overview', 'h2');
+        I.see('Example', 'h3');
+        I.see('Source Code', 'h3');
+        I.seeElement('code');
+        I.see('Variations', 'h3');
+        I.see('Props', 'h3');
+        I.seeElement('table');
+        I.see('property');
+        I.see('value');
+        I.see('required');
+        I.see('default');
+        I.seeNumberOfElements('td', 12 * 4);
+    });
+});
+
+Scenario('Avatar', (I) => {
+    I.searchWithinIframe(KIND, 'Overview', () => {
+        I.seeElement('.uir-avatar', 4);
```

Comment:
What does the `4` specify? I thought it was a count, but when I changed the value to `3` or `5`, the test still passed.

Reply:
Hmmm it should be a count!  `I.seeElement` needs to be swapped to `I.seeNumberOfElements` ... will change